### DISCUSSION TITLE
how to turn off "further_info_url" ?

### DISCUSSION THREAD

**Discussion by gleckler1** [PCMDI/cmor#719] [Q&A]
@durack1 @mauzey1 @manaster Following recommendations, we are trying to turn off the creation of this attribute for obs4MIPs but do not understand how/where it is set.   Any hints please?

**Comment by mauzey1** [comment]
There is currently no way of removing the attribute `further_info_url` from CMOR output.  If you want to replace the URL value of the attribute with a miscellaneous character (empty strings and whitespace throw an error), then you can set the value of `further_info_url` in your CV and user input JSON file to the same character.

In CV:
```
        "further_info_url":[
            "."
        ],
```
In user input:
```
    "further_info_url":       ".",
```
This will cause CMOR to output netCDF files with `further_info_url` set to `"."` instead of the generated URL.

**Comment by gleckler1** [comment]
Many thanks for the prompt response @mauzey1, this is a good solution for us.   Can you clarify where in the CV the "further_info_url" should go, e.g., in the CV below should it go within "version_metadata" or at the top level ? @durack1 @manaster

https://github.com/PCMDI/obs4MIPs-cmor-tables/blob/307_aim_gpcpv32/Tables/obs4MIPs_CV.json

**Reply by mauzey1** [reply]
Try adding the following within `CV` of the file.
```
        "further_info_url":[
            "."
        ],
```
